README.md

```diff
@@ -231,7 +232,8 @@ A curated list of projects powered by Noir.
 - [ZCaptcha](https://github.com/signorecello/zcaptcha) - A ZK version of Captcha
 - [ZKPic](https://github.com/0xrishabh/zkPic) - generate zero knowledge proofs for image edits
 - [Safecat](https://neimanslab.org/2024-02-19/safecat.html) - a simple CLI tool to generate, sign, and verify digital signatures using EdDSA Baby Jubjub Elliptic Curve signatures and a Poseidon hash function
-
+- [ZKEmail](https://github.com/zkemail/zkemail.nr) - a ZK Email version written in Noir. It is extremely performant on the client side, being ~2x faster on small proofs and ~5-10x speedups on the larger proofs, versus the Circom implementation of ZK Email.
```

Comment:
- Recommend listing it under "Miscellaneous", instead of "Signatures"
- Curious where are the speed comparisons with Circom sourced?